src/Tools/publish.ps1

```diff
@@ -0,0 +1,153 @@
+param([String]$version, [String]$apiKey, [String]$server, [String]$branchName, [String]$repoUrl, [String]$nugetRestoreAltSource = "", [bool]$pushTag, [String]$configuration, [String]$apiKeyInternal, [String]$internalServer, [String]$signUser = "", [String]$signSecret = "", $signConfigPath = "")

```

Comment:
All powershell scripts should be in `ci\scripts`. Just do not know how to differentiate them from framework scripts. What about adding folders or names of packages to the name of scripts?